### DISCUSSION TITLE
Extjs: unable to use 'title' property with 'Ext.TabPanel' in my code, but working fine in pimcore core files

### DISCUSSION THREAD

**Discussion by pradeeppimcore** [pimcore/pimcore#16427] [Q&A / Support]
We have made some custom screen using Ext.TabPanel in our bundle which are trying to upgrade to latest pimcore version.

'title' property with 'Ext.TabPanel' not working in my code, it does not break the js but it does not show the title on the tab panel either.
I looked at the Ext Js 7 docs and it seems that 'title' property with 'Ext.TabPanel' is not supported anymore.

But this property is working fine with the pimcore's core files

For example, the Users/Roles configuration screen. It uses the file "public/bundles/pimcoreadmin/js/pimcore/settings/user/usertab.js" which has the following code 

        this.panel = new Ext.TabPanel({
            title: this.data.user.name,
            closable: true,
            iconCls: "pimcore_icon_user",
            buttons: [{
                text: t("save"),
                handler: this.save.bind(this),
                iconCls: "pimcore_icon_accept"
            }]
        });

This code uses the 'title' property with 'Ext.TabPanel'. I am not able to understand how is this working with pimcore's code.
I will appreciate the help here as we are upgrading our bundle from older pimcore version.

**Comment by kingjia90** [comment]
> I looked at the Ext Js 7 docs and it seems that 'title' property with 'Ext.TabPanel' is not supported anymore.

i think it's inherited by the requirement of https://docs.sencha.com/extjs/7.0.0/modern/Ext.tab.Tab.html#cfg-title 


![image](https://github.com/pimcore/pimcore/assets/6014195/b937953b-f066-4dee-8a70-5d3f1fdc856c)
https://docs.sencha.com/extjs/7.0.0/modern/Ext.tab.Panel.html

**Reply by pradeeppimcore** [reply]
@kingjia90  I appreciate your quick reply, thanks for that.

I am assuming that if it is inherited than 'title' should work by default, but in our case it is not working.
I also tried this on the fiddle page of https://docs.sencha.com/extjs/7.0.0/modern/Ext.tab.Panel.html, if I give title to the Ext.tabPanel then it is not working. The title does not show up on the top of tab panel.

Whereas in the Extjs version 4 (https://docs.sencha.com/extjs/4.2.6/#!/api/Ext.tab.Panel) it is working fine.

Can you suggest on this.

**Reply by kingjia90** [reply]
I am not sure if i got correctly, and i am not expert in extjs but AFAIU `Ext.create('Ext.TabPanel')` is the tab-bar and there is no title in there, while `new Ext.TabPanel({` is the equivalent of creating a new item in the `Ext.create('Ext.TabPanel')` `items` config. In the example, `title: 'Home'` works, it is in the bottom area of the js fiddle
